admin_console/forms/PartnerConfiguration.php

```diff
@@ -150,16 +150,35 @@ public function init()
 				'label'			=> 'CDN Host:',
 				'filters'		=> array('StringTrim'),
 		));
-		
+
+
+		$this->addElement('checkbox', 'checkbox_cdn_api_host', array(
+			'decorators' => array('ViewHelper', array('Label', array('placement' => 'append')), array('HtmlTag',  array('tag' => 'dt', 'class' => 'partner_configuration_checkbox_field')))
+		));
+
+		$this->addElement('text', 'cdn_api_host', array(
```

Comment:
why do we need this? based on the URL protocol - http or https you can decide on the controller to which property to set this value.

Is there a use case where partner can have both HTTP and HTTPS different cdns?